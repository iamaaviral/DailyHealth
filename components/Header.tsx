import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

export default class Header extends React.Component<any,any> {

	render() {
		return (
			<View style={styles.header}>
				<Text style={styles.header_text}>Daily health</Text>
			</View>
		);
	}

}


const styles = StyleSheet.create({
	header: {
		flex: 1,
		flexDirection: 'column',
		alignSelf: 'stretch',
		paddingTop: 50,
		paddingBottom: 20,
		backgroundColor: 'rgba(0,170,170,0.1)'
	},
	header_text: {
		fontWeight: 'bold',
		fontSize: 25,
		textAlign: 'center'
	}
});