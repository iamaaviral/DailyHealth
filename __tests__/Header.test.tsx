// components/__tests__/Hello.tsx
import React from "react";
import "react-native";
import renderer from "react-test-renderer";

import Header from "../components/Header";

test("Header snapshot", () => {
  const snap = renderer.create(
    <Header />,
  ).toJSON();
  expect(snap).toMatchSnapshot();
});
