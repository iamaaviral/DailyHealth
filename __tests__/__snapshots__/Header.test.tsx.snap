// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<View
  style={
    Object {
      "alignSelf": "stretch",
      "backgroundColor": "rgba(0,170,170,0.1)",
      "flex": 1,
      "flexDirection": "column",
      "paddingBottom": 20,
      "paddingTop": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 25,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    Daily health
  </Text>
</View>
`;
